### PR TITLE
Fixing Travis issues with PR#57 (Add swap support)

### PR DESCRIPTION
**Pull request by dgoldsmith**

### CODE REVIEW

File: manifests/logical_volume.pp [hunk 1]
```diff
@@ -36,23 +63,39 @@
...
+        refreshonly => true,
+        subscribe   => Mount[$mount_title],
+      }
+    }
+  }
+  mount {"${mount_title}":
     ensure  => $mount_ensure,
-    device  => "/dev/${volume_group}/${name}",
+    name    => $fixed_mountpath,
+    device  => "${lvm_device_path}",
```
Comment (mhaskel) [inline manifests/logical_volume.pp]: This doesn't need to be quoted.

**Comment by mhaskel** [comment]
@dgoldsmith thanks for the contribution! Other than the one comment this looks good. This also needs to  be rebased against master. Thanks!

**Comment by tphoney** [comment]
ping @dgoldsmith could you please have a look at the comment @mhaskel made, and rebase.
Thanks !

**Comment by tphoney** [comment]
Thanks for the work you have put in to this pull request. However due to inactivity and it needs to be rebased. This pull request is being closed. If you want to re-submit this pr with the necessary changes, we will look at the new pull request.

**Pull request closed without merging**